src/main/java/org/folio/innreach/controller/UserCustomFieldMappingController.java

```diff
@@ -0,0 +1,44 @@
+package org.folio.innreach.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.folio.innreach.domain.service.UserCustomFieldMappingService;
+import org.folio.innreach.dto.UserCustomFieldMappingsDTO;
+import org.folio.innreach.rest.resource.UserCustomFieldMappingsApi;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.UUID;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/inn-reach/central-servers")
+public class UserCustomFieldMappingController implements UserCustomFieldMappingsApi {
+
+  private final UserCustomFieldMappingService service;
+
+  @Override
+  @GetMapping("/{centralServerId}/user-custom-field-mappings/{customFieldId}")
```

Comment:
JFYI, you can put this URI at the controller level as it is the same for both mappings and only use empty @GetMapping and @PutMapping